### PR TITLE
Add release instructions

### PR DESCRIPTION
**Pull request by leeky**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+I've used the 'gem-release' gem to simplify the process a little.
+Enter `gem install gem-release` to install on your local machine.
+
+## Release Process
+
+- Update `CHANGELOG.md` with any changes going into this release
+- Use `gem bump -v minor` to bump the minor release number of the gem
+- Use `gem tag` to create a git tag for the new version
+- Push the tag up to the git remote with `git push origin --tags`
+- Open a PR and get it merged into main
```
Comment (marianayovcheva) [inline RELEASE.md:17]: Potentially a daft question - when we start with the whole process, should we start off main, change the changelog.md there and then the gem takes care of creating a branch? Or should we branch off first?

Comment (leeky) [reply RELEASE.md:17]: I haven't really formalised the process yet, but generally I've been updating the CHANGELOG and bumping version numbers etc in whatever branch I'm currently working on.

Once that's merged to main, I add the release to the GitHub releases page.

Comment (marianayovcheva) [reply RELEASE.md:17]: Makes sense, for future reference are there cases it's okay to bundle multiple changes together or would you recommend one change per release? What I'm thinking about is that in design system we often have a few PRs grouped and then run a separate release process - the process is not perfect of course.

**Review by marianayovcheva — APPROVED** [review]

File: RELEASE.md (hunk 1 above)
Comment (leeky) [reply RELEASE.md:17]: I figured that once the form builder is feature-complete, we'll switch to bundling up changes together. Perhaps opening a release PR off main, just for the purpose of bumping the version number and pushing the tag.

**Pull request merged**